WcaOnRails/db/migrate/20220630233246_add_use_wca_live_for_scoretaking.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddUseWcaLiveForScoretaking < ActiveRecord::Migration[6.0]
+  def change
+    add_column :competitions, :use_wca_live_for_scoretaking, :boolean, null: false, default: true
+    Competition.update_all(use_wca_live_for_scoretaking: false)
```

Comment:
Right now you're updating _all_ competitions, even the ones that have been announced but haven't happened yet (i.e. future comps). Is this intended?

Reply:
Yes, the reason is there might be some competitions which do not use WCA Live. So, once the PR is merged, I'll connect with WCAT to get the current and future competitions to be updated and Jonathan to update the past competitions.